print/components/PicassoChunk.vue

```diff
@@ -240,13 +240,26 @@ export default {
 
 <style lang="scss">
 .landscape {
+  --font-scale: 0.67;
+  font-size: calc(10pt * var(--font-scale));
+
   transform: rotate(-90deg);
+  scale: 1.48;
   position: relative;
-  top: 320px;
+  top: 275px;
 
-  .fullwidth {
-    width: 960px;
-  }
+  width: 680px;
+  height: 459px;
+  overflow: visible;
+}
+
+.portrait {
+  --font-scale: 1;
+  font-size: 10pt;
+
+  width: 680px; /* 794px minus 114px (=2*15mm margin) */
```

Comment:
is it accidental that the width for portrait and landscape are the same?
If it should stay the same, then i would share the value

Reply:
Not accidental, see explanation on the other comment. With sharing you mean using a SCSS variable?

Reply:
using an scs variable or defining the width in a common css directive.
I don't know whic approach is best practice

Reply:
Implemented with https://github.com/ecamp/ecamp3/pull/3495/commits/ddc130919ee629bd9f33f54d5ff5ff9bb1296ae4